File: c++/Processor.cpp
class Processor{
	private:
		/*atribut private*/
		string name;
		string price;

	public:
		/*constructor*/
		Processor(){}
		Processor(string name, string price){
			this->name = name;
			this->price = price;
		}

		/*getter/setter*/
		void setname(string name){
			this->name = name;
		}
		string getname(){
			return this->name;
		}
		void setprice(string price){
			this->price = price;
		}
		string getprice(){
			return this->price;
		}

		/*Method*/
		void printProcessor(){
			cout << "Processor name : " << this->name << endl;
			cout << "Processor price : " << this->price << endl;
		}
		/*destructor*/
		~Processor(){}
};

File: php/Processor.php
<?php

class Processor{
	// atribut private
    private $name;
    private $price;

	// constructor
	public function __construct($name, $price){
        $this->name = $name;
        $this->price = $price;
	}

	// setter/getter
    public function setname($name){
		$this->name = $name;
	}
    public function getname(){
		return $this->name;
	}

    public function setprice($price){
		$this->price = $price;
	}
    public function getprice(){
		return $this->price;
	}

	/*Method*/
	public function printProcessor(){
		echo "Processor name : ".$this->name."<br>";
		echo "Processor price : ".$this->price."<br>";
	}
}

?>